### PR TITLE
w_timings tool

### PR DESCRIPTION
**Pull request by Daniel-Chem**
## Issue Number

Replaces PR #466.

## Describe the changes made
Cleaned up the `w_timings` tool’s output:
- Removed unnecessary decimal places from wallclock display  
- Added clear section headers (`===== WALLCLOCK =====`, `===== SIMULATION =====`, `===== RECYCLING =====`)  

Also added a new pytest in `tests/test_tools/test_w_timings.py` that:
- Mocks `argparse.Namespace` with the correct fields  
- Runs `entry_point()`  
- Asserts that each of the three section headers is printed  

## Goals and Outstanding Issues
- Finish the basic test for `w_timings` 
- Add numeric validation to verify the computed wallclock and simulation times  
- Extend tests to cover edge cases (e.g. zero iterations, custom `tau` values)

## Major files changed
- `src/westpa/cli/tools/w_timings.py`  
- `tests/test_tools/test_w_timings.py`  

## Status

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the CONTRIBUTING document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Additional context
Ran locally with:
```bash
pytest tests/test_tools/test_w_timings.py
# => 1 passed in 0.24s

### CODE REVIEW

File: tests/test_tools/test_w_timings.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from unittest import mock
+import argparse
+import pytest
+from westpa.cli.tools.w_timings import entry_point
+
+
+class Test_W_Timings:
+    """Test class for w_timings tool."""
+
+    @pytest.fixture
```
Comment (jpthompson17) [inline tests/test_tools/test_w_timings.py:11]: The `confest.py` file defines a [`ref_3iter`](https://github.com/westpa/westpa/blob/dda3e0b1ae8cf3446d7f3126295611e882096897/tests/conftest.py#L35) fixture, so there's no need for a separate fixture here. The `ref_3iter` fixture will  [set](https://github.com/westpa/westpa/blob/dda3e0b1ae8cf3446d7f3126295611e882096897/tests/conftest.py#L50) the test class's `h5_filepath` attribute,* which can then be used to set the `we_h5filename` field of the mock return value (see e.g. [`test_w_succ.py`](https://github.com/westpa/westpa/blob/dda3e0b1ae8cf3446d7f3126295611e882096897/tests/test_tools/test_w_succ.py#L20)).

*I'm not a big fan of this pattern of relying on a fixture to implicitly set test class attributes (what if we wanted to define test functions at the module level?), but it seems to be the way things are done at the moment.

File: tests/test_tools/test_w_timings.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import os
```
Comment (jpthompson17) [inline tests/test_tools/test_w_timings.py:1]: Unused import.

File: tests/test_tools/test_w_timings.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                count_events=True,
+            ),
+        ):
+            entry_point()
+
+        captured = capsys.readouterr()
+        output = captured.out
+
+        # Basic checks that output sections are present
+        # TODO: Replace with actual known values from reference
```
Comment (jpthompson17) [inline tests/test_tools/test_w_timings.py:38]: Please complete this TODO.

File: tests/test_tools/test_w_timings.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+class Test_W_Timings:
+    """Test class for w_timings tool."""
+
+    @pytest.fixture
+    def ref_west_file(self):
+        return "tests/refs/west_3iter.h5"
+
+    def test_run_w_timings(self, ref_west_file, capsys):
+        """
+        Currently only checks if sections headers are printing correctly
```
Comment (jpthompson17) [inline tests/test_tools/test_w_timings.py:17]: Remove this docstring after completing TODO below.

File: src/westpa/cli/tools/w_timings.py [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+        Returns
+        -------
+        int
+            Number of successful recycling events.
+        """
+        events = 0
+        # Get the key to the final iteration.
+        # Need to do -2 instead of -1 because there's an empty-ish final iteration written.
+        for iteration_key in tqdm(list(we_h5file['iterations'].keys())[-2:0:-1]):
```
Comment (jeremyleung521) [inline src/westpa/cli/tools/w_timings.py:58]: Might be more accurate to use -1 or -2 depending on 'walltime' in we_h5file['summary']. It'd be -1 if the file's been w_truncated, for example.

File: src/westpa/cli/tools/w_timings.py [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+            endpoint_types = we_h5file[f'iterations/{iteration_key}/seg_index']['endpoint_type']
+            if 3 in endpoint_types:
```
Comment (jeremyleung521) [inline src/westpa/cli/tools/w_timings.py:60]: import the Segment class and use the different status attributes instead. Much clearer that way than random integers.
https://github.com/westpa/westpa/blob/westpa2/src/westpa/core/segment.py#L24

File: src/westpa/cli/tools/w_timings.py [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    def go(self):
+        with self.data_reader:
+            we_h5file = self.data_reader.data_manager.we_h5file
+            self.iter_start = self.iter_range.iter_start
+            self.iter_stop = self.iter_range.iter_stop
+
+            walltime = we_h5file['summary']['walltime'][self.iter_start - 1 : self.iter_stop].sum()
+            aggtime = we_h5file['summary']['n_particles'][self.iter_start - 1 : self.iter_stop].sum()
+
+            days = int(walltime) // 86400
```
Comment (jeremyleung521) [inline src/westpa/cli/tools/w_timings.py:75]: Not sure why we aren't using `datetime.timedelta` like the rest of the WESTPA code
https://github.com/westpa/westpa/blob/westpa2/src/westpa/core/sim_manager.py#L751
